pangeo_forge_recipes/transforms.py

```diff
@@ -647,22 +611,37 @@ class StoreToZarr(beam.PTransform, ZarrWriterMixin):
       based on the full dataset (e.g. divide along a certain dimension based on a desired chunk
       size in memory). For more advanced chunking strategies, check
       out https://github.com/jbusecke/dynamic_chunks
-    :param dynamic_chunking_fn_kwargs: Optional keyword arguments for ``dynamic_chunking_fn``.
     :param attrs: Extra group-level attributes to inject into the dataset.
     :param encoding: Dictionary encoding for xarray.to_zarr().
     :param append_dim: Optional name of the dimension to append to.
+
+    Example of using a wrapper function to reduce the arity of a more complex dynamic_chunking_fn:
+
+        Suppose there's a function `calculate_dynamic_chunks` that requires extra parameters: an
+        `xarray.Dataset`, a `target_chunk_size` in bytes, and a `dim_name` along which to chunk.
+        To fit the expected signature for `dynamic_chunking_fn`, we can define a wrapper function
+        that presets `target_chunk_size` and `dim_name`:
+
+        ```python
+        def calculate_dynamic_chunks(ds, target_chunk_size, dim_name) -> Dict[str, int]:
+            ...
+
+        def dynamic_chunking_wrapper(ds: xarray.Dataset) -> Dict[str, int]:
+            target_chunk_size = 1024 * 1024 * 10
+            dim_name = 'time'
+            return calculate_dynamic_chunks(ds, target_chunk_size, dim_name)
+
+        StoreToZarr(..., dynamic_chunking_fn=dynamic_chunking_wrapper, ...)
+        ```
```

Comment:
Looks like this not rendering correctly. Not sure what your intended design is here, but maybe the paragraph starting with `Suppose..` should be dedented and the ```python fence is not necessary if you're indented?

See https://pangeo-forge--726.org.readthedocs.build/en/726/api_reference.html#ptransforms
<img width="788" alt="Screen Shot 2024-04-04 at 4 05 58 PM" src="https://github.com/pangeo-forge/pangeo-forge-recipes/assets/62192187/8af30b55-6970-407c-8050-6a46577119e4">